pkg/virt-handler/vm_test.go

```diff
@@ -1684,15 +1685,16 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(len(arg.(*v1.VirtualMachineInstance).Status.Interfaces)).To(Equal(1))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].Name).To(Equal(podCacheInterface.Iface.Name))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IP).To(Equal(podCacheInterface.PodIP))
-				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIP))
-
+				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIPs[0]))
+				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[1]).To(Equal(podCacheInterface.PodIPs[1]))
+				Expect(len(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs)).To(Equal(len(podCacheInterface.PodIPs)))
 			}).Return(vmi, nil)
 
 			controller.Execute()
 			Expect(len(controller.podInterfaceCache)).To(Equal(1))
 		})
 
-		It("Should update masquerade interface with the pod IP", func() {
+		table.DescribeTable("Should update masquerade interface with the pod IP", func(PodIPs []string) {
```

Comment:
the attribute name should start with lower case

Reply:
done (all the comments)